app/_hub/kong-inc/mocking/_index.md

```diff
@@ -103,76 +102,122 @@ params:
       value_in_examples: true
       description: |
         Randomly selects one example and returns it. This parameter requires the spec to have multiple examples configured.
+    - name: included_status_codes
+      required: false
+      default: null
```

Comment:
Is this a breaking change? It should default to `200`, `201` and `204`

Reply:
I don't think this is a breaking change. 
After upgrading to 3.1.x from 3.0.x, the mocking plugin will still select the minimum HTTP status code to return. 
Although there are `100` and `101` that is less than 200, but they cannot be used in ngx access phase.